src/site/includes/detail-page.html

```diff
@@ -10,37 +10,39 @@
 {% endcomment %}
 
 <main class="va-l-detail-page">
-  <div class="usa-grid usa-grid-full">
-    {% if hidesidenav === empty %}
-      {% include "src/site/components/navigation-sidebar.html" %}
-    {% endif %} 
-
-    <div class="usa-width-three-fourths">
-      {% if hidesidenav === empty %}
-        {% include "src/site/components/navigation-sidebar-trigger.html" with menuId = "va-detailpage-sidebar" %}
-      {% endif %}  
-      
-    {% if no_article_bottom_padding %}
-      <article class="usa-content vads-u-padding-bottom--0">
-        <h1>{{ heading }}</h1>
-        {{ contents }}
-      </article>
-    {% else %}  
-      <article class="usa-content">
-        <h1>{{ heading }}</h1>
-        {{ contents }}
-      </article>
-    {% endif %}
-
-      {% if majorlinks != empty %}
-        {% include "src/site/components/navigation-links-list.html" %}
+  {% if hidesidenav === empty %}
+    {% include "src/site/components/navigation-sidebar-trigger.html" with menuId = "va-detailpage-sidebar" %}
+  {% endif %}
+  <div class="vads-l-grid-container large-screen:vads-u-padding-x--0">
+    <div class="vads-l-row vads-u-margin-x--neg2p5">
+      <div class="vads-l-col--12 medium-screen:vads-u-padding-x--2 large-screen:vads-u-padding-x--2p5 medium-screen:vads-l-col--4 large-screen:vads-l-col--3">
+        {% if hidesidenav === empty %}
+          {% include "src/site/components/navigation-sidebar.html" with fullWidth = true %}
```

Comment:
Here's the param in the include.

Reply:
This is a temporary param so that we don't break the other layouts?

Reply:
Yes… but maybe the param name should be a little broader, like `newGrid`?